lib/sequelize.js

```diff
@@ -1200,6 +1202,7 @@ Sequelize.where = Sequelize.condition = Sequelize.prototype.condition = Sequeliz
 
  * @param {Object}   [options={}]
  * @param {Boolean}  [options.autocommit=true]
+ * @param {String}   [options.type='DEFERRED'] See `Sequelize.Transaction.TYPES` for possible options
```

Comment:
Might want to add a sqlite only notice here and the other place you mention it in the docs.